lib/cldr.js

```diff
@@ -1974,6 +1974,84 @@ Cldr.prototype = {
     );
 
     return languageData;
+  },
+
+  /**
+   * Weekdata contains various week related info, grouped by territory or locale
+   * Defaults are indicated by territory="001" or locale="und"
+   *
+   * https://www.unicode.org/reports/tr35/tr35-55/tr35-dates.html#Week_Data
+   */
+  extractWeekData() {
+
+    const finder = this.createFinder([
+      this.getDocument(
+        Path.resolve(
+          this.cldrPath,
+          'common',
+          'supplemental',
+          'supplementalData.xml'
+        )
+      )
+    ]);
+
+    const weekData = {
+      minDays: [],
+      firstDay: [],
+      weekendStart: [],
+      weekOfPreference: []
+    };
+
+    finder('/supplementalData/weekData/minDays').forEach(minDays => {
```

Comment:
Could you name the variable something like `minDaysNode` for clarity?

Would also be nice to transition towards `for...of`, but no biggie :)